plugin-server/src/cdp/templates/_destinations/native-webhook/webhook.template.ts

```diff
@@ -0,0 +1,91 @@
+import { NativeTemplate } from '~/cdp/templates/types'
+
+export const template: NativeTemplate = {
+    free: false,
+    status: 'hidden',
+    type: 'destination',
+    id: 'native-webhook',
```

Comment:
I don't love that we'd have this naming in the templates... Given they get written to the DB it all starts to feel a little messy. I'd really love it if we were actually moving to having a clear flag on the template kind to indicate what type it is as then we can keep the ids to `template-` prefixing.